packages/next/build/babel/preset.ts

```diff
@@ -155,7 +153,7 @@ module.exports = (
       !isServer && [
         require('@babel/plugin-transform-runtime'),
         {
-          corejs: isPolyfillsOptimization ? false : 2,
```

Comment:
Since this stops `@babel/plugin-transform-runtime` from polyfilling anything https://github.com/zeit/next.js/pull/9511 could be enabled again. All it would do then is reuse more helpers.

Reply:
Updating the version is moot now because the code doesn't even run.